File: 02_algorithm/programmers/Level2/문자열 내 p와 y의 개수.py
def solution(s):
    p_cnt, y_cnt = 0, 0
    for letter in s:
        if letter == 'p' or letter == 'P':
            p_cnt += 1
        elif letter == 'y' or letter == 'Y':
            y_cnt += 1
    if p_cnt == y_cnt:
        return True
    else:
        return False

print(solution('pPoooyY'))
print(solution('Pyy'))

File: 06_js/04_algorithm/programmers/Level1/문자열 내 p와 y의 개수.js
function solution(s){
    let answer = true;
    let p_cnt = 0;
    let y_cnt = 0;
    for (let i = 0; i < s.length; i++) {
        if (s[i] === 'p' || s[i] === 'P') {
            p_cnt += 1;
        } else if (s[i] === 'y' || s[i] === 'Y'){
            y_cnt += 1;
        }
    }
    if (p_cnt !== y_cnt) {
        answer = false;
    }
    return answer;
}